File: C++/factorial.cpp
#include<bits/stdc++.h>
using namespace std;
int factorial(int n){
    for(int i=n-1;i>=1;i--){
        n=n*i;
    }
    return n;
}
int main(){
    int n;
    cout<<"Enter the number: ";
    cin>>n;
    cout<<"The factorial of the given number is: "<<factorial(n);
    return 0;
}

File: Python/Factorial.py
a=input("Enter a number")
a=int(a)
f=1
if(a<0):
  print("Factorial does not exist")
elif(a==0):
  print("Factorial of 0 is 1")
else:
    for i in range(1,a+1):
      f=f*i
    print("Factorial is ",f)
